Cargo.toml

```diff
@@ -1,13 +1,26 @@
 [package]
 name = "dusk-abi"
-version = "0.1.0"
+version = "0.4.0"
 authors = ["Kristoffer Ström <kristoffer@dusk.network>"]
 edition = "2018"
 
+repository = "https://github.com/dusk-network/dusk-abi"
+keywords = ["cryptography", "crypto", "pki", "bls12-381"]
```

Comment:
Is this really correct? This looks like it's been copied from dusk-pki?

Reply:
No it's not correct. I thought I deleted it but apparently I commit an outdated `Cargo.toml`